lib/reindex/s3_bucketd.py

```diff
@@ -49,7 +51,7 @@ def inner(*args, **kwargs):
         return urllib.parse.quote(val.encode('utf-8'))
     return inner
 
-Bucket = namedtuple('Bucket', ['userid', 'name'])
+Bucket = namedtuple('Bucket', ['userid', 'name', 'object_locked'], defaults=[False])
```

Comment:
I was a bit confused, I suggest to rename `object_locked` to `object_lock_enabled`